src/main/java/seedu/address/model/person/Birthday.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class Birthday {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Birthdays should be in the form DD-MM-YYYY, and it should not be blank";
+
+    public static final String VALIDATION_REGEX = "^[0-9]{2}-[0-9]{2}-[0-9]{4}$";
```

Comment:
This validation allows invalid dates like `99-99-2021`. Perhaps you could check it with `DateTime` classes or use #41 `dateUtil`?